### PR TITLE
fix data order of tooltip

### PR DESCRIPTION
**Pull request by teichmaj**
fixes issue https://github.com/masayuki0812/c3/issues/1028

screenshot from 2015-08-19 16-33-34

screenshot from 2015-08-19 16-33-55

### CODE REVIEW

**Comment by teichmaj** [comment]
Now also with fixed tooltip order for stacked charts
screenshot from 2015-08-19 18-00-11
screenshot from 2015-08-19 18-00-14

File: src/tooltip.js [hunk 1]
```diff
@@ -29,7 +29,27 @@ c3_chart_internal_fn.getTooltipContent = function (d, defaultTitleFormat, defaul
...
+        text, i, title, value, name, bgcolor,
+	orderAsc = $$.isOrderAsc(), orderDesc = $$.isOrderDesc();
+     
+    if (config.data_groups.length === 0) {
+	d.sort(function(a,b){
+	    return orderAsc ? a.value - b.value : b.value - a.value;
+	});
+    } else {
+	var targets = $$.orderTargets($$.data.targets);
+	var data_targets = targets.map(function (i) {
```
Comment (masayuki0812) [inline src/tooltip.js]: Can you write like:
```
var ids = $$.orderTargets($$.data.targets).map(function (i) {
```
because it's not target, but ids.

**Comment by masayuki0812** [comment]
Can you fix travis and use space for indent, not tab?

**Comment by masayuki0812** [comment]
Thanks!

**Pull request merged**